lib/engine/game/g_1862_solo/step/buy_sell_par_shares.rb

```diff
@@ -7,7 +7,7 @@ module Game
     module G1862Solo
       module Step
         class BuySellParShares < Engine::Step::BuySellParShares
```

Comment:
Is there a reason you are not subclassing 1862's BuySellParShares?

Reply:
Good question. Not sure there was a reason, but I can try to do that. A quick browse did not see any reason why it should not work.

BTW, UNCHARTERED_TOKEN_COST exist in both 1862 buy_sell_par_shares and game.

Reply:
Have now done that in a new commit